From 909f94e79ead51cd999d19a1727bae44a42a423d Mon Sep 17 00:00:00 2001
From: Ihar Hrachyshka <ihrachys@redhat.com>
Date: Fri, 27 Mar 2015 17:13:37 +0100
Subject: [PATCH 2/2] Fixed rootwrap file installation

The previous patch [1] that introduced installation of configuration
files had a wrong order of arguments. That resulted in setuptools
installation failure as follows:

error: can't copy 'etc/designate/rootwrap/bind9.filters': doesn't exist
or not a regular file

The proper order is dest-dir = src-path, not vice versa.

[1]: Ib04ce7ca8e16ca1a07a39526c540802764a7c547

Conflicts:
	setup.cfg

Change-Id: Ie7924439eb33145339e0ada4433374ba54b9cd16
---
 setup.cfg | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/setup.cfg b/setup.cfg
index 8032877..ad476e4 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -33,8 +33,8 @@ data_files =
 	etc/designate/policy.json
 	etc/designate/designate.conf.sample
 	etc/designate/rootwrap.conf.sample
-	etc/designate/rootwrap.d =
-	etc/designate/rootwrap/bind9.filters
+	etc/designate/rootwrap =
+	etc/designate/rootwrap.d/bind9.filters
 
 [entry_points]
 console_scripts = 
-- 
2.1.0

